crates/sui-indexer/migrations/2023-08-19-044020_events/up.sql

```diff
@@ -12,6 +12,11 @@ CREATE TABLE events
     module                      text         NOT NULL,
     -- StructTag in Display format
     event_type                  text         NOT NULL,
+    -- Components of the StructTag: package, module, name
+    event_type_package          bytea        NOT NULL,
```

Comment:
for my info, how is `event_type_package` different from `package`?

Reply:
package is the package id where this event is emitted, which can be different from where the event type is defined.